docs/attributes-registry/README.md

```diff
@@ -82,6 +82,7 @@ Currently, the following namespaces exist:
 - [NodeJS](nodejs.md)
 - [OCI](oci.md)
 - [OpenTracing](opentracing.md)
+- [Oracledb](oracledb.md)
```

Comment:
please add acronym `OracleDB` to https://github.com/open-telemetry/semantic-conventions/blob/d3bff0dcb5dae5d1969f3d1dcced070cb3da88fd/templates/registry/markdown/weaver.yaml#L8

Reply:
Added it.